### PR TITLE
PLT-5071: Client side component of Telemetry.

### PR DESCRIPTION
**Pull request by grundleborg**
#### Summary
Client side component of Telemetry feature.

[Server PR](https://github.com/mattermost/platform/pull/5514)
[Docs PR](https://github.com/mattermost/docs/pull/912)

#### Ticket Link
https://mattermost.atlassian.net/browse/PLT-5071

#### Checklist
- [x] Includes text changes and localization file updates

### CODE REVIEW

File: webapp/components/webrtc/webrtc_controller.jsx [hunk 1]
```diff
@@ -747,6 +749,7 @@ export default class WebrtcController extends React.Component {
     }
 
     doHangup(error, manual) {
+        trackEvent('api', 'api_users_webrtc_end');
```
Comment (enahum) [inline webapp/components/webrtc/webrtc_controller.jsx]: shouldn't we track failed attempts?

Comment (grundleborg) [inline webapp/components/webrtc/webrtc_controller.jsx]: Where would you suggest putting the event for that?

Comment (enahum) [inline webapp/components/webrtc/webrtc_controller.jsx]: here? https://github.com/mattermost/platform/blob/master/webapp/components/webrtc/webrtc_controller.jsx#L602

Comment (grundleborg) [inline webapp/components/webrtc/webrtc_controller.jsx]: added.

Comment (enahum) [inline webapp/components/webrtc/webrtc_controller.jsx]: 👍

**Comment by crspeller** [comment]
@grundleborg So the segment tracking stuff doesn't go though the mattermost server, every client reports individually? How do we associate clients with particular servers?  

We should also be aware that some clients may not be able to report to an external service. Also some may use ad blockers to restrict segment in particular. This could be solved by going though the MM server. But it's probably not worth the extra traffic.

**Comment by grundleborg** [comment]
It doesn't go through the server because of the usual "performance reasons" - i.e. we don't want the server doing any extra work that isn't strictly necessary. See [here](https://github.com/mattermost/platform/pull/5516/files#diff-f1f7963b47ead3e8ded08b371f14420fR97) for how the client-side telemtry is linked back to the server side ones.

**Comment by crspeller** [comment]
Cool. Just a unique tracking ID. 

Going to set this as ready to merge so it can be merged with the server PR when it's done being reviewed.

**Pull request merged**